src/packagedcode/pypi.py

```diff
@@ -390,7 +390,11 @@ def assign_package_to_resources(cls, package, resource, codebase, package_adder)
         Assign files to package for an installed wheel. This requires a bit
         of navigation around as the files can be in multiple places.
         """
-        site_packages = resource.parent(codebase).parent(codebase)
+        dist_info_dir = resource.parent(codebase)
+        if not dist_info_dir:
+            return
+
+        site_packages = dist_info_dir.parent(codebase)
         if not site_packages:
             return
```

Comment:
Let also check that this is a proper site-packages dir and that this match the name pattern.
```suggestion
            return
        if not site_packages.name == "site-packages":
            return
```

Reply:
Adding thanks.